src/main.css

```diff
@@ -7,10 +7,6 @@
   --red: #db6b77;
 }
 
-::-webkit-calendar-picker-indicator {
-  filter: invert(1);
-}
```

Comment:
This was causing all the calendar datepicker icons to disappear (was done for dark mode originally)